src/modules/projects/projects.service.ts

```diff
@@ -1,19 +1,28 @@
 import { Injectable } from '@nestjs/common';
-import { NotFoundException } from '@nestjs/common/exceptions';
+import {
+  ForbiddenException,
+  NotFoundException,
+} from '@nestjs/common/exceptions';
 import { plainToInstance } from 'class-transformer';
+import { API_RESPONSE_MESSAGES } from 'src/constants';
 import { Req, Res } from 'src/dtos';
 import {
   ProfessionalsProjectsRepository,
   ProjectsRepository,
 } from 'src/repositories';
+import { ProjectAttributesRepository } from 'src/repositories/project-attributes.repository';
 import { helperCreatePaginatedResponse } from 'src/utils/api-paginated-response';
+import { isSameOrAfter, isSameOrBefore } from 'src/utils/date-validations';
```

Comment:
```suggestion
import { helperCreatePaginatedResponse,  isSameOrAfter, isSameOrBefore } from 'src/utils';
```
We need to export those files on src/utils/index.ts